discount/src/main/java/com/kyriba/conference/discount/util/DiscountDataMapper.java

```diff
@@ -0,0 +1,35 @@
+package com.kyriba.conference.discount.util;
+
+import com.kyriba.conference.discount.api.dto.DiscountExternalDto;
+import com.kyriba.conference.discount.api.dto.DiscountResponse;
+import com.kyriba.conference.discount.api.dto.DiscountUpdateParamsDto;
+import com.kyriba.conference.discount.domain.DiscountEntity;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+
+/**
+ * @author Igor Lizura
+ */
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface DiscountDataMapper {
```

Comment:
Mapping is very simple in your case, so library is not needed here.

Reply:
I just played with this tool

Reply:
Each tool is applicable and useful only in some context. :)